docs/documentation/release_notes/topics/24_0_0.adoc

```diff
@@ -195,6 +195,32 @@ spec:
       key: config.xml
 ----
 
+= Keycloak CR resources options
+
+The Keycloak CR now allows for specifying the `resources` options for managing compute resources for the Keycloak container.
+It provides the ability to request and limit resources for the inner Keycloak CR and KeycloakRealmImport CR container.
+
+You can specify your custom values based on your requirements as follows:
+
+----
+apiVersion: k8s.keycloak.org/v2alpha1
+kind: Keycloak
+metadata:
+  name: example-kc
+spec:
+  ...
+  resources:
+    requests:
+      cpu: 1200m
+      memory: 896Mi
+    limits:
+      cpu: 6
+      memory: 3Gi
+----
+
+For more details, check the
+https://www.keycloak.org/operator/advanced-configuration[Operator Advanced configuration].
```

Comment:
I'm not sure if there's a way to specify even the section(newly created) and have the ExternalLinksTest pass :) 

Keeping it as is for now. 

We should also reorder the new things for KC 24 doc, as all changes to Keycloak CR should be somewhere together (follow-up issue).